packages/cyberstorm/src/components/LinkLibrary.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  LinkLibrary,
+  thunderstoreLinkProps,
+  ThunderstoreLinkProps,
+} from "@thunderstore/components";
+import NextLink from "next/link";
+import React, { forwardRef } from "react";
+
+interface CyberstormLinkProps {
+  children?: React.ReactNode;
+}
+
+interface LinkProps extends CyberstormLinkProps, ThunderstoreLinkProps {
+  queryParams?: string;
+  url: string;
+}
+
+const CyberstormLink = forwardRef<HTMLAnchorElement>(
+  (props: CyberstormLinkProps, ref) => {
+    const { ...rest } = props;
+
+    return (
+      // eslint-disable-next-line jsx-a11y/anchor-has-content
+      <a ref={ref} {...rest} />
+    );
+  }
+);
+
+CyberstormLink.displayName = "CyberstormLink";
+
+const Link = (props: LinkProps): React.ReactElement => {
+  const { children, queryParams, url, ...cyberStormProps } = props;
+  const q = queryParams ? `?${queryParams}` : "";
+
+  for (const key in thunderstoreLinkProps) {
+    delete cyberStormProps[key as keyof ThunderstoreLinkProps];
+  }
+
+  return (
+    <NextLink href={`${url}${q}`} passHref>
```

Comment:
To reiterate earlier comments, neither `@thunderstore/cyberstorm` nor `@thunderstore/cyberstorm-storybook` uses Next.js or its link component.